lesson 2. exercise 2.py

```diff
@@ -15,13 +15,17 @@
     "12": 'декабря',
 }
 dict_days = {
-    '01': 'первое', '02': 'второе', '03': 'третье', '04': 'четвёртое', '05': 'пятое',
-    '06': 'шестое', '07': 'седьмое', '08': 'восьмое', '09': 'девятое', '10': 'десятое',
-    '11': 'одиннадцатое', '12': 'двенадцатое', '13': 'тринадцатое', '14': 'четырнадцатое', '15': 'пятнадцатое',
-    '16': 'шестнадцатое', '17': 'семнадцатое', '18': 'восемнадцатое', '19': 'девятнадцатое', '20': 'двадцатое',
-    '21': 'двадцать первое', '22': 'двадцать второе', '23': 'двадцать третье', '24': 'двадцать четвёртое',
-    '25': 'двадцать пятое', '26': 'двадцать шестое', '27': 'двадцать седьмое', '29': 'двадцать девятое',
-    '28': 'двадцать восьмое', '30': 'тридцатое', '31': 'тридцать первое',
+    '01': 'первое', '02': 'второе', '03': 'третье', '04': 'четвёртое',
+    '05': 'пятое', '06': 'шестое', '07': 'седьмое', '08': 'восьмое',
+    '09': 'девятое', '10': 'десятое', '11': 'одиннадцатое',
+    '12': 'двенадцатое', '13': 'тринадцатое', '14': 'четырнадцатое',
+    '15': 'пятнадцатое', '16': 'шестнадцатое', '17': 'семнадцатое',
+    '18': 'восемнадцатое', '19': 'девятнадцатое', '20': 'двадцатое',
+    '21': 'двадцать первое', '22': 'двадцать второе', '23': 'двадцать третье',
+    '24': 'двадцать четвёртое', '25': 'двадцать пятое',
+    '26': 'двадцать шестое', '27': 'двадцать седьмое',
+    '29': 'двадцать девятое', '28': 'двадцать восьмое',
+    '30': 'тридцатое', '31': 'тридцать первое',
```

Comment:
dana_data = '03.11.2019'
and
dana_data = '3.11.2019'
return different resutls

Reply:
I resolved this mistake